cmd/earthly/project_cmds.go

```diff
@@ -350,6 +350,8 @@ func (app *earthlyApp) projectOrgName(ctx context.Context, cloudClient *cloud.Cl
 
 	if app.orgName != "" {
 		return app.orgName, nil
+	} else if app.cfg.Global.Org != "" {
+		return app.cfg.Global.Org, nil
```

Comment:
the `else` is unnecessary here.
```suggestion
	}
	if app.cfg.Global.Org != "" {
		return app.cfg.Global.Org, nil
```